submodules/usersCallForwarding/views/complexStrategy.html

```diff
@@ -0,0 +1,184 @@
+<div class="complex-strategy-content" data-strategy="{{strategy}}">
+    <div class="complex-strategy-header">
+        <div class="forward-from">
+            <label>
+                Forward calls from
+            </label>
+
+            <div class="complex-rule">
+                {{#monsterRadio @root.i18n.users.callForwarding.strategies.selective.all }}
```

Comment:
You don't need the `@root` in this case,
```suggestion
                {{#monsterRadio i18n.users.callForwarding.strategies.selective.all }}
```